src/NoteMouseHandler/NoteMouseHandler.js

```diff
@@ -0,0 +1,97 @@
+function findMap(arr, func) {
+  for (const item of arr) {
```

Comment:
__[jshint]__
*  'for of' is available in ES6 (use 'esversion
*  'const' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->